ext/pgsql/tests/bug77047.phpt

```diff
@@ -19,12 +19,16 @@ pg_query($db, "CREATE TABLE bug77047 (
         t TIME WITHOUT TIME ZONE
     )");
 
-pg_insert($db, "bug77047", array("t" => "13:31"));
-pg_insert($db, "bug77047", array("t" => "13:31:13"));
-pg_insert($db, "bug77047", array("t" => "1:2:3"));
-pg_insert($db, "bug77047", array("t" => "xyz"));
-pg_insert($db, "bug77047", array("t" => NULL));
-pg_insert($db, "bug77047", array("t" => ""));
+try {
+    pg_insert($db, "bug77047", array("t" => "13:31"));
```

Comment:
You're only testing one case now, because when pg_insert (expectedly) fails, it will execute the catch and not execute the other pg_insert calls.